cmake/README.ProjectCiFileChangeLogic.md

```diff
@@ -0,0 +1,14 @@
+The file ProjectCiFileChangeLogic.py in this directory is only a symlink to
+the real file under:
+
+```
+  commonTools/framework/ProjectCiFileChangeLogic.py
+```
+
+That is because that is under the base directory for the
+`TrilinosFrameworkTests` TriBITS package.  We want changes to the file
+`ProjectCiFileChangeLogic.py` to trigger the enable of the
+TrilinosFrameworkTests package so as to run the unit tests for that module.
+But the file `ProjectCiFileChangeLogic.py` also needs to exist in this this
+directory so that TriBITS tools will find it and use it automatically.  Using
+a symbolici link satisifes both of these requirements.
```

Comment:
Spelling error: "symbolic" (OK to leave for now and fix in next commit)

Reply:
@mhoemmen, I fixed these in PR #3169.  I would have pushed these to this PR but I don't want to risk the PR builds failing and blocking the merge of this PR.